packages/variant-theme/package.json

```diff
@@ -2,13 +2,17 @@
   "name": "@fluentui-contrib/variant-theme",
   "version": "0.0.1",
   "dependencies": {
-    "@swc/helpers": "~0.5.2"
+    "@swc/helpers": "~0.5.2",
+    "@fluentui/react-components": ">=9.46.3 <10.0.0",
+    "@fluentui/react-migration-v8-v9": "^9.6.12",
+    "@fluentui/scheme-utilities": "^8.3.46",
```

Comment:
these thw packages need to be installed within monorepo root, otherwise things will not work ( we use single version policy - https://opensource.google/documentation/reference/thirdparty/oneversion )

`yarn add -DW @fluentui/scheme-utilities @fluentui/react-migration-v8-v9`

Reply:
fixed